libraries/AP_Scripting/applets/Script_Controller.lua

```diff
@@ -184,12 +183,11 @@ function mission_load(n)
   local index = 0
   local fail = false
   while true and not fail do
-     local data = {}
      local line = file:read()
      if not line then
         break
      end
-     local ret, _, seq, curr, frame, cmd, p1, p2, p3, p4, x, y, z, autocont = string.find(line, "^(%d+)%s+(%d+)%s+(%d+)%s+(%d+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+(%d+)")
+     local ret, _, seq, _, frame, cmd, p1, p2, p3, p4, x, y, z, _ = string.find(line, "^(%d+)%s+(%d+)%s+(%d+)%s+(%d+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+(%d+)")
```

Comment:
```suggestion
     local ret, _, seq, _ --[[ curr ]]--, frame, cmd, p1, p2, p3, p4, x, y, z, _ --[[ autocont ]]-- = string.find(line, "^(%d+)%s+(%d+)%s+(%d+)%s+(%d+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+([-.%d]+)%s+(%d+)")
```

A bit clunky, but would stop the field definitions getting lost to time? We could also call them `_curr` etc but that would introduce more names.

Reply:
done